doc/user/PARSING.md

```diff
@@ -67,7 +67,20 @@ When formatting a vector into a string, individual token are formatted according
 
 ## Color
 
-Color are parsed and formatted as a vector of double.
+The following formats are supported when parsing a color, case insensitive:
+
+- R,G,B where R, G, B are doubles [0, 1]
+- #RRGGBB where RR, GG, BB are hexadecimal values
+- rgb(R, G, B) where R, G and B are integer [0, 255]
+- hsl(H, S%, L%) where H is integer [0, 360], S and L are integer [0, 100]
+- hsv(H, S%, V%) where H is integer [0, 360], S and V are integer [0, 100]
+- hwb(H, W%, B%) where H is integer [0, 360], W and B are integer [0, 100]
+- cmyk(C%, M%, Y%, K%) where C, M, Y, K are integer [0, 100]
+- [color name](https://htmlpreview.github.io/?https://github.com/Kitware/vtk-examples/blob/gh-pages/VTKNamedColorPatches.html)
+
+See [W3C](https://www.w3.org/TR/css-color-3/#rgb-color) doc for more details on these formats.
+
+When formatting a color into a string, it is formatted as `#RRGGBB` if values are multiple of 255. Otherwise, it is formatted as `R,G,B` where R, G, B are doubles between 0 and 1.
```

Comment:
```suggestion
When formatting a color into a string, it is formatted as `#RRGGBB` if values are multiple of 255. Otherwise, it is formatted as vector of doubles.
```